pkgs/applications/video/vlc/default.nix

```diff
@@ -39,7 +40,7 @@ stdenv.mkDerivation rec {
     libkate libtiger libv4l samba liboggz libass libdvbpsi libva
     xorg.xlibsWrapper xorg.libXv xorg.libXvMC xorg.libXpm xorg.xcbutilkeysyms
     libdc1394 libraw1394 libopus libebml libmatroska libvdpau libsamplerate
-    fluidsynth wayland wayland-protocols
+    fluidsynth wayland wayland-protocols protobuf
```

Comment:
`nativeBuildInputs` maybe because protoc is executed at build time?

Reply:
Good idea, unsure.  It does retain a dep via libtool files it looks like?

```
$ PAGER= nix why-depends -f . vlc protobuf --all                                                       14:33:09 on 19-01-02
/nix/store/a7y3ssv1dmv4dhv2qjxczybjr3ia1ds0-vlc-3.0.5
╚═══lib/vlc/plugins/stream_out/libstream_out_chromecast_plugin.la: … -lpthread -lsystemd /nix/store/bscqmp3iisv2liwa7gdrnvs9paixak6a-protobuf-3.6.1/lib/libprotobuf-…
    lib/vlc/plugins/stream_out/libstream_out_chromecast_plugin.so: …dbus-1.12.12-lib/lib:/nix/store/bscqmp3iisv2liwa7gdrnvs9paixak6a-protobuf-3.6.1/lib:/nix/store/b…
    => /nix/store/bscqmp3iisv2liwa7gdrnvs9paixak6a-protobuf-3.6.1
```

Reply:
ok. Probably it would be both in the cross-compiling case then.